cmd/build.go

```diff
@@ -64,6 +65,7 @@ func Build() *cobra.Command {
 	cmd.Flags().StringVarP(&tag, "tag", "t", "", "name and optionally a tag in the 'name:tag' format (it is automatically pushed)")
 	cmd.Flags().StringVarP(&target, "target", "", "", "set the target build stage to build")
 	cmd.Flags().BoolVarP(&noCache, "no-cache", "", false, "do not use cache when building the image")
+	cmd.Flags().BoolVarP(&plain, "plain", "", false, "show plain build container output")
```

Comment:
I'd use the same format as buildkit: `--progress=plain` and `--progress=tty` as the default?